_includes/tutorials/multi-joins/ksql/markup/dev/create-inputs-customers.adoc

```diff
@@ -0,0 +1,6 @@
+First let's create an input https://docs.ksqldb.io/en/latest/concepts/collections/#tables[Table] of customer data which will hold data in JSON format over 1 partition:
```

Comment:
Is it significant that these are all 1 partition?  If yes, please explain why.  If no, consider omitting to make the example simpler.

Reply:
removed in latest